modules/cnvkit/batch/main.nf

```diff
@@ -19,25 +19,42 @@ process CNVKIT {
     }
 
     input:
-    tuple val(meta), path(tumourbam), path(normalbam)
+    tuple val(meta), path(tumorbam), path(normalbam)
```

Comment:
sorry for being picky, but since it also takes cram file, how about renaming to `tumor_input` etc?

Reply:
No worries - it's a good idea. I put tumor and normal. That also better aligns with the input switch names (tumor and normal), Similarly target_file is now targets to match with the `CNVkit` switch name